node/internal/actors/os_procs.go

```diff
@@ -112,11 +114,16 @@ type logCapture struct {
 
 // Log capture implementation of io.Writer for stdout and stderr
 func (cap logCapture) Write(p []byte) (n int, err error) {
+	e := models.NewEnvelope[string](WorkloadLogType, string(p), 200, nil)
```

Comment:
I don't think we need an envelope here. We should just have a standard structure for log entries. The envelope from models has things like a result code which don't apply to logs.